src/conf/local-file.yaml

```diff
@@ -39,6 +39,8 @@ observability:
   system-observability: true
   network-observability: false
   write-logs-to-db: true
+  cron-job-time-for-purge-db: "10d0h0m00s"
+  purge-logs-from-db: true
```

Comment:
```suggestion
  purge-olds-db-entries: true
```

Reply:
Accepted and Thank you for suggesting and reviewing.